akka-http-core/src/main/scala/akka/http/impl/engine/parsing/HttpRequestParser.scala

```diff
@@ -32,10 +32,10 @@ private[http] final class HttpRequestParser(
   import HttpMessageParser._
   import settings._
 
-  val in = Inlet[SessionBytes]("HttpRequestParser.in")
-  val out = Outlet[RequestOutput]("HttpRequestParser.out")
```

Comment:
let's keep in/out in classes where we have simple Flows - it's simpler, no need to add the full types into the names